pmm-app/src/pmm-qan/panel/components/Details/Metadata/Metadata.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+import { Overlay } from 'shared/components/Elements/Overlay/Overlay';
+import { Collapse, useStyles2 } from '@grafana/ui';
+import { Table } from '../../../../../shared/components/Elements/Table';
+import { LinkTooltip } from '../../../../../shared/components/Elements/LinkTooltip/LinkTooltip';
+import { getStyles } from './Metadata.styles';
+import { metadataTableHead } from './Metadata.constants';
+
+const Metadata = ({ metadata, loading }) => {
+  const styles = useStyles2(getStyles);
+  const [isMetadataOpen, setMetadataVisibility] = useState(true);
+
+  const nameColumn = (item) => (
+    <span className={styles.metadataColumn}>
+      <span>{item.name}</span>
+    </span>
+  );
+
+  const valueColumn = (item) => (
+    <span className={styles.metadataColumn}>
+      <span>{item.value}</span>
+      <LinkTooltip tooltipText={item.tooltip} icon="info-circle" />
+    </span>
+  );
+
+  const columns = [
+    {
+      Header: metadataTableHead.first,
+      accessor: nameColumn,
+    },
+    {
+      Header: metadataTableHead.second,
+      accessor: valueColumn,
+    },
+  ];
+
+  return (
+    <Overlay size={35}>
```

Comment:
why 35?

Reply:
I've kept the style of the table 'Metrics' above to have the same size.